context/src/main/java/io/opentelemetry/context/propagation/ContextPropagators.java

```diff
@@ -71,25 +73,29 @@
 public interface ContextPropagators {
 
   /**
-   * Returns a {@link ContextPropagatorsBuilder} used to construct a new {@code ContextPropagators}
-   * object with the specified propagators.
-   *
-   * <p>Invocation order of {@code TextMapPropagator#inject()} and {@code
-   * TextMapPropagator#extract()} for registered trace propagators is undefined.
-   *
-   * <p>This is a example of a {@code ContextPropagators} object being created:
+   * Returns a {@link ContextPropagators} which can be used to extract and inject context in text
+   * payloads with the given {@link TextMapPropagator}. Use {@link
+   * TextMapPropagator#composite(TextMapPropagator...)} to register multiple propagators, which will
+   * all be executed when extracting or injecting.
    *
    * <pre>{@code
-   * ContextPropagators propagators = DefaultContextPropagators.builder()
-   *     .addTextMapPropagator(new HttpTraceContext())
-   *     .addTextMapPropagator(new HttpBaggage())
-   *     .addTextMapPropagator(new MyCustomContextPropagator())
-   *     .build();
+   * ContextPropagators propagators = ContextPropagators.create(
+   *   TextMapPropagator.composite(
+   *     HttpTraceContext.getInstance(),
+   *     W3CBaggagePropagator.getInstance(),
+   *     new MyCustomContextPropagator()));
```

Comment:
this usage looks really good to me.  👍

Reply:
I preferred the `Builder` but hiding the default instance looks great. On that note, `TraceMultiPropagator` _may_ need to be lifted too, i.e. no `Builder`.

Reply:
Thanks for calling out - let me followup with consistency in `TraceMultiPropagator` in a bit